doc/nrf/ug_nrf9160.rst

```diff
@@ -110,9 +110,18 @@ See the `Mobile network operator certifications`_ for more information.
 There are two ways to update the modem firmware:
 
 Full upgrade
-  A full upgrade of the modem firmware requires a wired connection.
-  The upgrade is done through the nRF Connect Programmer, which is part of `nRF Connect for Desktop`_.
-  See `Updating the nRF9160 DK cellular modem`_ in the nRF Connect Programmer User Guide for instructions.
+  Both a wired or a wireless connection are suitable to proceed to a full upgrade of the modem firmware:
+
+  * When using a wired connection, you can use either the *nRF Connect Programmer*, as part of `nRF Connect for Desktop`_, or a custom application.
+    Both methods leverage the Simple Management Protocol (SMP) to provide an interface over UART which enables the device to perform the update.
+
+    The nRF Connect Programmer can be used to perform the update, regardless of the existing firmware.
+    See `Updating the nRF9160 DK cellular modem`_ in the nRF Connect Programmer user guide for more details.
+
+    See the :ref:`fmfu_smp_svr_sample` for a sample meant to showcase the integration of the :ref:`lib_fmfu_mgmt` subsystem in a custom application.
+
+  * When using a wireless connection, the upgrade is applied as firmware over-the-air (FOTA).
```

Comment:
"is applied over the air", or "is applied as firmware over-the-air (FOTA) upgrade."

Reply:
fixed